src/commands/cli/schemas/collect.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as path from 'path';
+import * as os from 'os';
+import { FlagsConfig, SfdxCommand } from '@salesforce/command';
+import { fs, Messages } from '@salesforce/core';
+import { cp } from 'shelljs';
+import * as fg from 'fast-glob';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-release-management', 'cli.schemas.collect');
+
+export class SchemaUtils {
+  public static async getLatestSchemaFiles(): Promise<string[]> {
+    const pjson = (await fs.readJsonMap(path.join(process.cwd(), 'package.json'))) as { oclif: { plugins: string[] } };
+    const globs = (pjson.oclif?.plugins || []).map((plugin) => {
+      const normalized = plugin.replace(/\\/g, '/');
+      return `node_modules/${normalized}/schemas/**/*.json`;
+    });
+    const schemaFiles = (await fg(globs)).filter((f) => !f.includes(path.join('@salesforce', 'schemas')));
+    return schemaFiles;
+  }
+
+  public static async getExistingSchemaFiles(): Promise<string[]> {
+    const globs = ['schemas/**/*.json'];
```

Comment:
To go along with the `outputDir` comment below.
```suggestion
    const globs = ['pluginSchemas/**/*.json'];
```

Reply:
I'd like to keep it `schemas` so that it's consistent with our plugins